src/ServerConfigParams.ts

```diff
@@ -71,7 +71,12 @@ export interface ServerConfigParams {
      * @note the minus sign is omitted if used
      * @note fractions exceeding the number of decimals of that of the native currency will be rounded up
      */
-    feePercentage: string;
+    feePercentage?: string;
```

Comment:
I'm not sure we need to make this optional. I think we could just set a default value if they're not specified, the same way as we do for `feesReceiver`.

Reply:
updated